src/components/Controls.vue

```diff
@@ -138,8 +138,30 @@
 								<label :for="user.uid"><NcAvatar :user="user.uid" :size="24" :disable-menu="true" /> {{ user.displayname }}</label>
 							</div>
 
-							<h3>{{ t('deck', 'Filter by due date') }}</h3>
+							<h3>{{ t('deck', 'Filter by done') }}</h3>
+							<div class="filter--item">
+								<input id="open"
```

Comment:
Nitpick: Maybe make the id more unique as just `open` has a bit of conflict potential. Maybe `filter-option-open`